src/docker.test.ts

```diff
@@ -90,16 +131,19 @@ describe("Docker images", (): void => {
     expect(docker.CACHE_HIT).toBe("cache-hit");
   });
 
+  test("exports DOCKER_IMAGES_LIST", (): void => {
+    expect(docker.DOCKER_IMAGES_LIST).toBe("docker-images-list");
+  });
+
   test("exports DOCKER_IMAGES_PATH", (): void => {
     expect(docker.DOCKER_IMAGES_PATH).toBe("~/.docker-images.tar");
   });
 
   test("are loaded on cache hit", async (): Promise<void> => {
     await mockedLoadDockerImages(true);
 
-    expect(util.execBashCommand).lastCalledWith(
-      `docker load --input ${docker.DOCKER_IMAGES_PATH}`
-    );
+    expect(core.saveState).toHaveBeenCalledTimes(1);
+    expect(util.execBashCommand).toHaveBeenCalledTimes(1);
```

Comment:
We can actually move this one assert to `assertLoadDockerImages` and perform it unconditionally.